source/module_md/test/FIRE_test.cpp

```diff
@@ -0,0 +1,116 @@
+#include "setcell.cpp"
```

Comment:
It is not recommended to include a cpp file.

Reply:
OK, I will update it.